tests/installation/bootloader_zkvm.pm

```diff
@@ -40,7 +40,14 @@ sub set_svirt_domain_elements {
             $cmdline .= " live.password=$testapi::password";
             # add extra boot params for agama network, e.g. ip=2c-ea-7f-ea-ad-0c:dhcp
             $cmdline .= ' ' . get_var('AGAMA_NETWORK_PARAMS') if get_var('AGAMA_NETWORK_PARAMS');
-            $cmdline .= ' ' . 'inst.dud=' . data_url(get_var('INST_DUD')) . ' rd.neednet=1' if get_var('INST_DUD');
+
+            # additional parameters requiring parsing
+            if (get_var('INST_DUD')) {
+                foreach my $item (split(',', get_var('INST_DUD'))) {
+                    $cmdline .= ' inst.dud=' . data_url($item);
+                }
+                $cmdline .= ' rd.neednet=1';
+            }
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details